lib/peer.go

```diff
@@ -312,7 +312,7 @@ func (pp *Peer) HelpHandleInv(msg *MsgDeSoInv) {
 			// For transactions, check that the transaction isn't in the
 			// mempool and that it isn't currently being requested.
 			_, requestIsInFlight := pp.srv.requestedTransactionsMap[currentHash]
-			if requestIsInFlight || pp.srv.mempool.IsTransactionInPool(&currentHash) {
+			if requestIsInFlight || pp.srv.GetMempool().GetTransaction(&currentHash) != nil {
```

Comment:
Can we use `IsTransactionInPool` instead of `GetTransaction` here? Anywhere we're checking if a transaction exists in the pool, but don't need access to the transaction, we should use `IsTransactionInPool`.

Reply:
We certainly can update this. I didn't realize `IsTransactionInPool` was part of the Mempool interface! I like this more streamlined version, too.